spec.html

```diff
@@ -8274,7 +8274,14 @@
         <h1>[[DefineOwnProperty]] ( _P_, _Desc_ )</h1>
         <p>When the [[DefineOwnProperty]] internal method of a module namespace exotic object _O_ is called with property key _P_ and Property Descriptor _Desc_, the following steps are taken:</p>
         <emu-alg>
-          1. Return *false*.
+          1. Let _current_ be ? _O_.[[GetOwnProperty]](_P_).
+          1. If _current_ is *undefined*, return *false*.
+          1. If IsAccessorDescriptor(_Desc_) is *true*, return *false*.
+          1. If _Desc_.[[Writable]] is present and has value *false*, return *false*.
```

Comment:
Would this perhaps be cleaner (and independently useful) if there was an `IsSameDescriptor` abstract operation? It might also be helpful in cleaning up `ValidateAndApplyPropertyDescriptor` as well.